### PR TITLE
fix strict type CompanyTypeClient

### PR DESCRIPTION
**Pull request by gengjozsef**

### CODE REVIEW

File: src/FondOfSpryker/Client/CompanyType/Zed/CompanyTypeStub.php [hunk 1]
```diff
@@ -21,6 +22,7 @@ public function __construct(CompanyTypeToZedRequestClientInterface $zedRequestCl
     }
 
     /**
+     *
```
Comment (daniel-rose) [inline src/FondOfSpryker/Client/CompanyType/Zed/CompanyTypeStub.php]: remove line (phpcs)

File: src/FondOfSpryker/Zed/CompanyType/Business/Model/CompanyTypeReaderInterface.php [hunk 2]
```diff
@@ -5,11 +5,13 @@
 use Generated\Shared\Transfer\CompanyCollectionTransfer;
 use Generated\Shared\Transfer\CompanyTransfer;
 use Generated\Shared\Transfer\CompanyTypeCollectionTransfer;
+use Generated\Shared\Transfer\CompanyTypeResponseTransfer;
 use Generated\Shared\Transfer\CompanyTypeTransfer;
 
 interface CompanyTypeReaderInterface
 {
     /**
+     *
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/CompanyType/Business/Model/CompanyTypeReaderInterface.php]: remove line (phpcs)

**Review by daniel-rose — CHANGES_REQUESTED** [review]
remove deprecated stuff

File: src/FondOfSpryker/Client/CompanyType/Zed/CompanyTypeStub.php (hunk 1 above)
Comment (gengjozsef) [reply src/FondOfSpryker/Client/CompanyType/Zed/CompanyTypeStub.php]: done

File: src/FondOfSpryker/Zed/CompanyType/Business/Model/CompanyTypeReaderInterface.php (hunk 2 above)
Comment (gengjozsef) [reply src/FondOfSpryker/Zed/CompanyType/Business/Model/CompanyTypeReaderInterface.php]: done

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**